packages/website/builder/config.json

```diff
@@ -40,10 +40,25 @@
       "../../../docs/api/action",
       "../../../docs/api/module",
       "../../../docs/api/providers",
-      "../../../docs/api/components",
       "../../../docs/api/compute",
       "../../../docs/api/universalcontroller"
     ],
+    "views": [{
+      "name": "react",
+      "path": "../../node_modules/@cerebral/react/README"
+    }, {
+      "name": "vue",
+      "path": "../../node_modules/@cerebral/vue/README"
+    }, {
+      "name": "inferno",
+      "path": "../../node_modules/@cerebral/inferno/README"
+    }, {
+      "name": "preact",
+      "path": "../../node_modules/@cerebral/preact/README"
+    }, {
+      "name": "angularjs",
+      "path": "../../node_modules/@cerebral/angularjs/README"
+    }],
     "addons": [{
       "name": "router",
       "path": "../../node_modules/@cerebral/router/README"
```

Comment:
Could also sort these ones while we are here...

```
firebase
firebase-admin
forms
http
router
shortcuts
storage
useragent
```

Reply:
Agreed, updated :) Removed `function-tree`, I think it deserves its own "thing". Not sure what, will think on it